deploy.go

```diff
@@ -467,6 +468,17 @@ func (d *imageDeployer) run(opts *deployOptions) error {
 		return xerrors.Errorf("cannot obtain source file information: %w", err)
 	}
 
+	rootPartitionUUIDOverride := opts.RootPartitionUUID != ""
+	espPartitionUUIDOverride := opts.ESPPartitionUUID != ""
+
+	if rootPartitionUUIDOverride != espPartitionUUIDOverride {
+		return errors.New("Please override both root and efi partititions")
```

Comment:
The language should suggest that both overridden and neither overridden are the two acceptable states; the current wording only captures the former.